identity/src/main/java/com/android/identity/ConnectionMethod.java

```diff
@@ -105,7 +105,7 @@ ConnectionMethod fromNdefRecord(@NonNull NdefRecord record, boolean isForHandove
 
         // NFC Carrier Configuration record
         //
-        if (record.getTnf() == 0x02
+        if (record.getTnf() == NdefRecord.TNF_EXTERNAL_TYPE
```

Comment:
This is a technical change (TNF_EXTERNAL_TYPE is 0x04) and I think this is correct.